bakery/src/tasks/check-feed.js

```diff
@@ -0,0 +1,45 @@
+const dedent = require('dedent')
+
+const task = ({ awsAccessKeyId, awsSecretAccessKey, bucketName, feedFileUrl }) => {
```

Comment:
I think this makes sense, but I just want to confirm - every distribution pipeline targets it's own bucket, correct?

Reply:
Yes, that is indeed my working assumption, but I did add an acceptance criteria to [cnx #938](https://github.com/openstax/cnx/issues/938) where we should explicitly come to a decision on this.